### ISSUE TITLE
option for triggered departure in vehroute output

### ISSUE THREAD

**Issue by rummel123** [eclipse/sumo#8803]
For persons in the vehroute output, the departure should not use "triggered" but the departure time.
With option "--vehroute-output.intended-depart" "triggerd" should be used.

**Comment by namdre** [comment]
@behrisch If we change the default to "actual departure time" scenarios with triggered persons and vehicles cannot be re-run unless option **--vehroute-output.intended-depart** is set (there won't be an error but some rides will never start).

Would it be better to add another attribute to the vehroute output ('started') and leave the depart attribute as it is?

**Comment by behrisch** [comment]
Another idea is to invent the option **--vehroute-output.actual-depart** which has somehow the inverted semantics of intended-depart and deprecate the latter. This would solve both ways of using it without another attribute,

**Comment by namdre** [comment]
Right now, actual depart times are already written for all non-triggered vehicles and the option  **--vehroute-output.intended-depart**  changes this behavior.  It's only the 'triggerd' vehicles that always maintain their 'intended' value.  If we want to solve this with backward compatible options instead of a new attribute, we need a new option just for the triggered vehicles that can be freely combined with the older option.

**Comment by behrisch** [comment]
Yes, but I think the current behavior of writing actual depart times for non-triggered but "triggered" for triggered vehicles is really odd. The only reason I can think of to keep it is really backward compatibility. Is this worth it?

**Comment by namdre** [comment]
It's worth it, because otherwise, the vehroute-output cannot be re-used as simulation input (vehicles and persons that were connected via `triggered` in the original input will not start their rides. Hence this would be a breaking change.

With the new option, users may think they can use **--vehroute-output.actual-depart** to re-run the simulation (and save on all the wasted computational effort from delayed insertion) and then wonder why the rides don't start.
